openspoor/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .transformers.TransformerCoordinatesToSpoor import *
```

Comment:
keep using relative imports from within the python package itself to itself.
With absolute imports of itself, you can get in all kinds of issues, revolving around circular imports.